helm/bits/values.yaml

```diff
@@ -18,6 +18,9 @@ global:
     rootfs_downloader: eirini/rootfs-downloader:2.32.0@sha256:6ae511688a27a453dcf31bf5a3bd7287ba99233e1586c7aeb78c87a18c68dbe4
   labels: {}
   annotations: {}
+  resources:
```

Comment:
Does this make sense under `global`? Wouldn't resource requirements of, say, UAA be vastly different from that of `bits`?  Note that `global.images.bits_service` for example really scopes it down to just `bits`.  Of course, I'm not convinced `rootfs_version` makes any sense here either…

Reply:
AFAICT in the `kubecf` chart the `global` here is put under the `bits` key, limiting the scope.
See https://github.com/cloudfoundry-incubator/kubecf/pull/1014/files

Reply:
I guess I could make `resources` its own toplevel key.

Reply:
Done with commit 935c4a4f960b8b455d6c76fb727c1a65cab05f6d.